metadata_mapper/lambda_function.py

```diff
@@ -3,7 +3,7 @@
 from urllib.parse import urlparse, parse_qs
 import settings
 import importlib
-from mappers.abstract_mapper import UCLDCWriter, AbstractRecord, AbstractVernacular
+from mappers.mapper import UCLDCWriter, Record, Vernacular
```

Comment:
Thanks @timfrazee - we've decided module name `mapper` and class names `Record` and `Vernacular` are what we would like to use (as you have here).